tutorials/custom-migration-tutorial.md

```diff
@@ -0,0 +1,109 @@
+- `hash_key_name`: The custom contract package hash NamedKey as a String. See the [ARG_HASH_KEY_NAME_1_0_0](https://github.com/casper-ecosystem/cep-78-enhanced-nft/blob/408db77c3b9ca22752c7f877ea99a01dfca03a7b/contract/src/main.rs#L2006).
+- `access_key_name`: The custom contract package access NamedKey as a String. See the [ARG_ACCESS_KEY_NAME_1_0_0](https://github.com/casper-ecosystem/cep-78-enhanced-nft/blob/408db77c3b9ca22752c7f877ea99a01dfca03a7b/contract/src/main.rs#L2005).
+
+Here is the `casper-client` command to upgrade and migrate to version 1.1.1 when custom NamedKeys are used:
+
+```bash
+casper-client put-deploy \
+--node-address [NODE_SERVER_ADDRESS] \
+--chain-name [CHAIN_NAME] \
+--secret-key [KEY_PATH]/secret_key.pem \
+--payment-amount [PAYMENT_AMOUNT_IN_MOTES] \
+--session-path [PATH]/contract.wasm \
+--session-arg "named_key_convention:u8='1'" \
+--session-arg "collection_name:string='[COLLECTION_NAME]'" \
+--session-arg "hash_key_name:string='[ARG_HASH_KEY_NAME_1_0_0]'" \
+--session-arg "access_key_name:string='[ARG_ACCESS_KEY_NAME_1_0_0]'" 
+```
+
+Here is the full list of required arguments:
+- `node-address`: An IP address of a peer on the network. The default port for JSON-RPC servers on Mainnet and Testnet is 7777.
+- `chain-name`: The chain name of the network where you wish to send the deploy. For Mainnet, use *casper*. For Testnet, use *casper-test*.
+- `secret-key`: The file name containing the secret key of the account paying for the deploy.
+- `payment-amount`: The payment for the deploy in motes.
+- `session-path`: The path to the compiled Wasm on your computer. When using the [cep-78-wasm.tar.gz](https://github.com/casper-ecosystem/cep-78-enhanced-nft/releases/download/v1.1.1/cep-78-wasm.tar.gz) provided, this would be the path to the `contract.wasm` file.
+- `named_key_convention`: Argument that specifies the use of the `V_1_0_standard` [NamedKeyConvention](../README.md#namedkeyconventionmode).
+- `collection_name`: Argument that specifies the collection name as a String.
+- `hash_key_name`: The custom contract package hash NamedKey as a String.
+- `access_key_name`: The custom contract package access NamedKey as a String.
+
+The command returns the deploy hash that you can use to verify whether or not the deploy succeeded.
+
+**Important Notes**: 
+
+- Suppose an account attempts to install a second CEP-78 contract instance with the same collection name, but a custom contract package access NamedKey is used. In this case, the installation will not overwrite the NamedKey entry under which the access URef is written.
+- When installing a new version of the CEP-78 contract, you do not need to specify all the runtime arguments needed during the initial installation of version 1.0.0, such as `total_token_supply`, `ownership_mode`, etc. 
```

Comment:
```suggestion
- Suppose an account attempts to upgrade by installing a second CEP-78 contract instance with the same collection name, but a custom contract package access NamedKey is used. In this case, the installation will not overwrite the NamedKey entry under which the access URef is written.
- When upgrading by installing a second a CEP-78 contract, you do not need to specify all the runtime arguments needed during the initial installation of version 1.0.0, such as `total_token_supply`, `ownership_mode`, etc. 
```

Reply:
I should remove the first point because it is confusing. I wanted to explain why custom NamedKeys are used. But it isn't another upgrade path. If I add "upgrade" in there, it will get confusing. Whoever is on this tutorial knows why they used custom NamedKeys.

I will keep the second point and suggestion by saying: "When upgrading by installing a new version of the CEP-78 contract...".  Because it might be a 3rd upgrade (v1.0.0, v1.1.0, v1.1.1).

Reply:
I have removed the first point from the standard path as well, to avoid confusion. The readme is already clear enough.